coherence/v1requests_test.go

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2024 Oracle and/or its affiliates.
+ * Licensed under the Universal Permissive License v 1.0 as shown at
+ * https://oss.oracle.com/licenses/upl.
+ */
+
+package coherence
+
+import (
+	"github.com/onsi/gomega"
+	"testing"
+)
+
+func TestNewNamedCacheRequest(t *testing.T) {
```

Comment:
What is this test actually doing?

Reply:
This was a mistake and should have been deleted. Done